base/exports.jl

```diff
@@ -177,20 +180,25 @@ export
     ÷,
     &,
     *,
+    ×,
```

Comment:
This operator is already exported by LinearAlgebra for the cross product, so we can't define it in Base.

Reply:
Are they incompatible? I mean, it seems the cross product of vectors won’t interfere with the same types of operands, or did I miss something?

Reply:
Yes, you can for example multiply the adjoint of a vector with a regular vector. I would also be very surprised if `A × B` was a matrix product. I don't think we should conflate those two meanings.

Reply:
Yes, `×` definitely shouldn't just be an alias for `*`, that would be mathematically surprising.